api/src/queries/survey/survey-update-queries.ts

```diff
@@ -5,6 +5,78 @@ import { generateGeometryCollectionSQL } from '../generate-geometry-collection';
 
 const defaultLog = getLogger('queries/survey/survey-update-queries');
 
+/**
+ * SQL query to update a permit row based on an old survey association.
+ *
+ * @param {number} surveyId
+ * @returns {SQLStatement} sql query object
+ */
+export const putOldSurveyPermitNumberSQL = (surveyId: number): SQLStatement | null => {
```

Comment:
Can we teak this naming to something more like: "putUnsetSurvetPermitSQL" or "putRemoveSurveyPermitSQL", etc.  Just so its more clear what its doing.  Same goes for the function doc, being explicit about what this function does exactly, since it has a pretty specific purpose.

Reply:
Actually, we really don't even need the 'put', 'post', etc, prefixes as these aren't really related to http requests.  They could just be "removeSurveyPermitSQL" or "unassociatePermitFromSurveySQL"

Reply:
The names of these functions, in general, is probably something we could do a tech-debt pass on at some point.  Making sure the names are informative, and the jsdoc is detailed.